corehq/apps/hqadmin/utils.py

```diff
@@ -175,3 +178,19 @@ def unset_password(user):
     # 128 bits / 8 (bits/byte) = 16 bytes
     random_key = os.urandom(16).hex()
     user.set_password(random_key)
+
+
+def get_download_url(content, name, content_type=None, timeout=24 * 60):
```

Comment:
open to suggestions on the names I used here for the util, management command, and download view - none of them really feel perfect to me.